source/manual/republishing-content.html.md

```diff
@@ -8,18 +8,32 @@ parent: "/manual.html"
 
 Sometimes it may be necessary to republish content to make it show up on the website. For example if we make an update to [govspeak][govspeak-repo] that would require us to re-render and save new HTML for content.
 
-This process varies per app.
+This process varies per app and requires
+
+- Connection to the [VPN][vpn] and
+- [Production access][production-access]
 
 ## Whitehall
 
-If the document is in Whitehall, there is a Rake task you can run.
+If the document is in Whitehall, there is a Rake task you can run:
 
 [`publishing_api:republish_document[slug]`][republish-whitehall-doc-jenkins]
 
 For organisations, run:
 
 [`publishing_api:republish_organisation[slug]`][republish-whitehall-org-jenkins]
 
+for document type, run:
+
+[`publishing_api:document_type[slug]`][republish-whitehall-document-type-jenkins]
```

Comment:
Could you update both instances to say 
publishing_api:document_type[document_class] 
since we're not using slugs here and it's a bit confusing.

Reply:
Perrfect, DocumentClass makes more much sense. 🙌🏼